client/gutenberg/extensions/publicize/form-unwrapped.jsx

```diff
@@ -75,10 +86,14 @@ class PublicizeFormUnwrapped extends Component {
 				</label>
 				<div className="jetpack-publicize-message-box">
 					<textarea
-						value={ shareMessage }
-						onChange={ messageChange }
+						value={ this.getShareMessage() }
+						onChange={ this.onMessageChange }
 						disabled={ this.isDisabled() }
 						maxLength={ MAXIMUM_MESSAGE_LENGTH }
+						placeholder={ __(
+							"Write a message for your audience here. If you leave it blank, the post's title will be used."
```

Comment:
What about setting the placeholder to the post title? This would somewhat imply the behavior that this string is now describing. (It would also reflect the Classic Editor's Publicize module's behavior.)

Reply:
> What about setting the placeholder to the post title?

(I.e. to `defaultShareMessage`)

Reply:
I think this may cause a little frustration for first time publicize users. Sure, you and I would understand the directness of displaying the post title when the field is blank, because we use publicize all the time. Someone who is using Publicize for the first time would appreciate a bit more information describing the situation.